src/scrabble.js

```diff
@@ -1,6 +1,70 @@
 class Scrabble {
-  // Write your implementation here
+  constructor(word) {
+    this.word = word;
+  }
 
+  score() {
+    if (typeof this.word !== "string" || this.word.length === 0) {
+      return 0;
+    }
+    this.word = this.word.toLowerCase();
+    const points1 = ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"];
+    const points2 = ["d", "g"];
+    const points3 = ["b", "c", "m", "p"];
+    const points4 = ["f", "h", "v", "w", "y"];
+    const points5 = ["k"];
+    const points8 = ["j", "x"];
+    const points10 = ["q", "z"];
+
+    let score = 0;
+    let scoreMultiplier = 1;
+
+    for (let i = 0; i < this.word.length; i++) {
+      // decide if double or tripple letter
+      if (this.word[i] === "{") {
+        scoreMultiplier = 2;
+      } else if (this.word[i] === "}") {
+        scoreMultiplier = 1;
+      } else if (this.word[i] === "[") {
+        scoreMultiplier = 3;
+      } else if (this.word[i] === "]") {
+        scoreMultiplier = 1;
```

Comment:
You could consider improving this section to be able to score double/triple words with double/triple letters within. The way it is now, if you try to score {[w]ord}, once the ']' is detected your multiplier is set to 1 when the letters 'ord' should still be scored x2.